examples/README.md

```diff
@@ -75,11 +75,11 @@ OTEL_DOTNET_AUTO_LEGACY_SOURCES="InstrumentedHttpCall.GetAsync" exampleApp=Bindi
 
 ### ConsoleApp
 
-The default example app used by `./run-example.sh`. It contains manual instrumentation using an
+The default sample app used by `./run-example.sh`. It contains manual instrumentation which use an
```

Comment:
@Kielek per comment in the other PR about the manual instrumentation example, the info about manual instrumentation on the ConsoleApp (default example app) should be removed.